### PR TITLE
FE-1093 - Form field components should have a baked-in max-width

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-1093](https://jira.int.messagesystems.com/browse/FE-1093)

### What Changed

- Added wrapping `<Box/>` that can receive the `maxWidth` prop - defaulting to the `1200` design token
- Removed feature-by-feature setting of max width in favor of global component default prop
- Fixed a few spots in which the default max width needed an override

### How To Test

- Click through the app with Hibana enabled, visually verify forms
- Ensure most fields have the max width set
- Ensure there aren't unusual spots in which the max width should be set to something else!

### To Do
- [ ] Incorporate feedback

### CODE REVIEW

**Comment by nicklemmon** [comment]
Failing integration test is a result of some tests that should be skipped for Hibana.

**Comment by shrirupa** [comment]
We should may be wrap the TypeAheadWrapper in a Box with maxWidth 
Screen Shot 2020-06-09 at 3 08 55 PM

File: src/pages/suppressions/components/AddForm.js [hunk 1]
```diff
@@ -44,23 +44,19 @@ export class AddForm extends Component {
...
-              </Box>
+              <Field
+                name="recipient"
+                component={TextFieldWrapper}
+                validate={[required, email]}
+                required
+                label="Email Address"
+              />
+              <Field
+                component={SubaccountTypeaheadWrapper}
```
Comment (shrirupa) [inline src/pages/suppressions/components/AddForm.js:55]: https://github.com/SparkPost/2web2ui/pull/1652#issuecomment-641516844

File: src/pages/suppressions/components/UploadForm.js [hunk 2]
```diff
@@ -54,14 +54,12 @@ export class UploadForm extends Component {
...
               />
-              <Box maxWidth="1200">
-                <Field
-                  component={SubaccountTypeaheadWrapper}
-                  disabled={submitting}
-                  helpText="Leaving this field blank will add the suppressions to the master account."
-                  name="subaccount"
-                />
-              </Box>
+              <Field
```
Comment (shrirupa) [inline src/pages/suppressions/components/UploadForm.js:57]: https://github.com/SparkPost/2web2ui/pull/1652#issuecomment-641516844

File: src/pages/trackingDomains/components/CreateForm.js [hunk 3]
```diff
@@ -15,24 +14,20 @@ export class CreateForm extends Component {
...
+              <Field
+                component={TextFieldWrapper}
+                label="Domain Name"
+                name="domain"
+                // Do not try to validate tracking domains, let our API make that decision
+                validate={[required]}
+                disabled={submitting}
+              />
+              <Field
+                component={SubaccountTypeaheadWrapper}
```
Comment (shrirupa) [inline src/pages/trackingDomains/components/CreateForm.js:26]: https://github.com/SparkPost/2web2ui/pull/1652#issuecomment-641516844

**Comment by nicklemmon** [comment]
@shrirupa Ready for another pass on this one - I think I got it.

**Pull request merged**